addons/event/static/src/views/event_registration_kanban_controller.js

```diff
@@ -0,0 +1,58 @@
+import { kanbanView } from "@web/views/kanban/kanban_view";
+import { KanbanController } from "@web/views/kanban/kanban_controller";
+import { EventRegistrationSummaryDialog } from "@event/client_action/event_registration_summary_dialog";
+import { onMounted, onWillUnmount } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+export default class EventRegistrationKanbanController extends KanbanController {
+
+    setup() {
+        super.setup()
+        this.dialog = useService("dialog");
+        this.orm = useService("orm")
+        onMounted(this.hideSystray);
+        onWillUnmount(this.showSystray);
+    }
+
+    async openRecord(record, mode) {
+        if (!this.props.context.active_model) {
```

Comment:
This is to exclude the cases we are coming from another model I guess? It feels strange to have different conditions in both controllers to 'enter this mode'. Could we instead check if some value is True? Or even be sure we can use it in this view with controlling the places we use this view? (Open question)

Reply:
Fixed it! by passing an additional context :smile: